backend/chainlit/callbacks.py

```diff
@@ -16,7 +16,7 @@
 
 @trace
 def password_auth_callback(
-    func: Callable[[str, str], Awaitable[Optional[User]]]
+    func: Callable[[str, str], Awaitable[Optional[User]]],
```

Comment:
Why the `,` here?

Reply:
That is the result of running `ruff format`.